search/search_service/api/document_update.py

```diff
@@ -0,0 +1,44 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from http import HTTPStatus
+
+from flask_restful import Resource, request
+from flasgger import swag_from
+
+from amundsen_common.models.search import UpdateDocumentRequestSchema
+from search_service.proxy import get_proxy_client
+from search_service.proxy.es_search_proxy import RESOURCE_STR_MAPPING
+
+
+class DocumentAPI(Resource):
+
+    def __init__(self) -> None:
+        self.proxy = get_proxy_client()
+        self.request = UpdateDocumentRequestSchema().loads(json.dumps(request.get_json()))
```

Comment:
Accept add or overwrite option to explicitly decide the behavior, either append a value so there are multiple values for a field or overwrite it completely

Reply:
Support multiple values per field so [] instead of just oen value

Reply:
Support updating multiple fields in one single request, {tag: ['t1', 't2'], description: 'i am a table'}

Reply:
ignore last 2, double check the behavior for why we default to empty array for badges and tags, does ES support multi value in a field by default?